src/swoop/api/routers/processes.py

```diff
@@ -112,20 +127,16 @@ async def get_process_description(
     workflows = (
         request.app.state.workflows.get("workflows")
         if request.app.state.workflows.get("workflows")
-        else []
+        else {}
```

Comment:
I don't think this is right. I think it is just `request.app.state.workflows`, which will always be a dict.